EIPS/eip-6206.md

```diff
@@ -35,8 +35,8 @@ A new instruction, `JUMPF (0xb2)`, is introduced.
 Let the definition of `type[i]` be inherited from [EIP-4750](./eip-4750.md) and define `stack_height` to be the height of the stack at a certain instruction during the instruction flow traversal if the operand stack at the start of the function were equal to `type[i].inputs`.
 
 * The immediate argument of `JUMPF` MUST be less than the total number of code sections.
-* The stack height at `JUMPF` MUST be greater than or equal to `type[code_section_index].inputs`.
```

Comment:
Old version: 
1. `stack_height >= type[code_section_index].inputs`
2. `type[current_section_index].outputs == stack_height - type[code_section_index].inputs + type[code_section_index].outputs`

Point 2 gives requirement on exact value of `stack_height`, therefore `stack_height` can be expressed in terms of sections inputs/outputs.